src/hrtf/neighbors.c

```diff
@@ -53,11 +53,11 @@ MYSOFA_EXPORT struct MYSOFA_NEIGHBORHOOD *mysofa_neighborhood_init(struct MYSOFA
 				break;
 			}
 			phi += 0.5;
-		} while (phi <= 45);
+		} while (phi2 <= lookup->phi_max);
```

Comment:
I would add one more interation here just to be save...
phi2 <= lookup->phi_max  + 0.5

Reply:
wise